python/delayed-pi-exchange/pireceiver.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import helics as h
+
+fedinitstring = "--federates=1"
+deltat = 0.01
+
+helicsversion = h.helicsGetVersion()
+
+print("PI RECEIVER: Helics version = {}".format(helicsversion))
+
+# Create Federate Info object that describes the federate properties */
+print("PI RECEIVER: Creating Federate Info")
+fedinfo = h.helicsFederateInfoCreate()
+
+# Set Federate name
+print("PI RECEIVER: Setting Federate Info Name")
+status = h.helicsFederateInfoSetFederateName(fedinfo, "TestB Federate")
+
+# Set core type from string
+print("PI RECEIVER: Setting Federate Info Core Type")
+status = h.helicsFederateInfoSetCoreTypeFromString(fedinfo, "zmq")
+
+# Federate init string
+print("PI RECEIVER: Setting Federate Info Init String")
+status = h.helicsFederateInfoSetCoreInitString(fedinfo, fedinitstring)
+
+# Set the message interval (timedelta) for federate. Note that
+# HELICS minimum message time interval is 1 ns and by default
+# it uses a time delta of 1 second. What is provided to the
+# setTimedelta routine is a multiplier for the default timedelta.
+
+# Set one second message interval
```

Comment:
you look like it is setting a 0.01 s deltat,  if you wanted interval you should use the period property.  either the comment or the value should change here.